syntax="proto2";

package aap_protobuf.service.control.message;

message AuthResponse {
    required int32 status = 1;
}
